auspice-client/customisations/navbar.js

```diff
@@ -122,8 +123,8 @@ const NavBar = ({sidebar, narrativeTitle, width}) => {
           null : (
             <div style={{...styles.flexColumns, paddingRight: "12px"}}>
               <div style={{flex: 5}}/>
-              <Link href="https://docs.nextstrain.org/en/latest/index.html" style={styles.link} text="DOCS"/>
-              <Link href="https://docs.nextstrain.org/en/latest/learn/about-nextstrain.html" style={styles.link} text="HELP"/>
```

Comment:
Threading [this suggestion](https://github.com/nextstrain/nextstrain.org/pull/735#pullrequestreview-1727893206):

> The "Docs" and "Help" links are still in the Auspice navbar, but we should probably condition those away too:
> 
> ![image](https://github.com/nextstrain/nextstrain.org/assets/79913/522f4fe3-e826-429d-9ba8-2887a3f986d9)

Reply:
I can think of two ways to address this:

1. Conditionally render the links.
2. Make this part of the navbar customizable.

I tried (1) in 0e291625fcb0db34b7bd0db68f80e590d0640c79...521d00bf9e6c163b1e79c0bbffe33d90dae35711, however am not very satisfied with this approach.

(2) seems to introduce unnecessary nextstrain.org-specific customization on top of the auspice customization. I haven't tried it yet, but I suppose I should since it would parallel the approach used in static-site.

Reply:
Hmm, yeah, not a fan of (1) since it's a dynamic condition based on a request instead of a static build-time condition.

(2) would be better.

We can also punt on this for now.

Reply:
In 8e27349cbe337869ea1d03335a5f259741137411 I've chosen to leave it up to the user:

https://github.com/nextstrain/nextstrain.org/blob/8e27349cbe337869ea1d03335a5f259741137411/groupsApp.md?plain=1#L24-L29